lib/_stream_writable.js

```diff
@@ -666,6 +666,10 @@ function finishMaybe(stream, state, sync) {
 
 function finish(stream, state) {
   state.pendingcb--;
+  if (state.errorEmitted) {
+    return;
+  }
```

Comment:
```suggestion
  if (state.errorEmitted)
    return;
```

(side note, I really wish JavaScript had `return if (state.errorEmitted)` syntax ;-) ...)